src/lib/prov/tpm2/tpm2_ecc/info.txt

```diff
@@ -0,0 +1,17 @@
+<defines>
+TPM2_ECC_ADAPTER -> 20240925
+</defines>
+
+<module_info>
+name -> "TPM2 ECC Adapter"
+brief -> "Support for RSA ECC pairs hosted on TPM 2.0"
+</module_info>
+
+<requires>
+ecdsa
+ecdh
```

Comment:
When following through with [disentangling `tpm2_crypto_backend` and the availability of `tpm2_rsa`/`tpm2_ecc`](https://github.com/Rohde-Schwarz/botan/pull/14#issuecomment-2391322414), we should remove this (currently fairly surprising) requirement.